src/components/navbar.component.tsx

```diff
@@ -56,6 +65,43 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
     };
   }, []);
 
+  let [searchBarIsVisible, setSearchBarIsVisible] = React.useState<any>(false);
+
+  let [closeSearchBar, setCloseSearchBar] = React.useState<any>(false);
```

Comment:
Actually, `closeSearchBar` doesn't look like it's used for anything. Which is good, because "closeSearchBar" is a verb, and therefore doesn't make sense as a state variable name. I think you can just delete that line and all the references to it.

Reply:
c@donaldkibet  you suggested i change the `  const [searchBarIsVisible, setSearchBarIsVisible]` however it's value is depended upon by `setSearchBar` for it's change of value.

Reply:
`searchBarIsVisible` is a React state and the appropriate way to change its value is using the `setSearchBarIsVisible` function. you can check [useState](https://reactjs.org/docs/hooks-reference.html)